settings.gradle

```diff
@@ -25,3 +25,5 @@ include ':autodispose'
 include ':autodispose-kotlin'
 include ':sample'
 include ':test-utils'
+include 'autodispose-rxlifecycle-interop'
```

Comment:
Let's call this just `autodispose-rxlifecycle`, and make the package name `com.uber.autodispose.rxlifecycle`.